src/public/script.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
```

Comment:
Do not disable any eslint rule

Reply:
This is the file for the front-end notification test that we said we will be deleting. Because some values are gotten in the connected Html, then it is hard for me to resolve the eslint errors

Reply:
Prefix each variable with `window.` because all declared variables on the browser are scoped to the window object. Moreover, that has nothing to do with the eslint rule you disabled. You shouldn't have console.log in your code by the way.

Reply:
Console.log was removed. but even if I set every variable. `window.` the error with eslint still come